### ISSUE TITLE
Add CSS to the project

### ISSUE THREAD

**Issue by MattCrl** [MattCrl/Contributors-portraits#2]
You can submit your CSS ideas for this project, everything that is not ugly will be welcome !

**Comment by 94151** [comment]
@MattCrl i want to work on this issue. can you guide me to start on

**Comment by rahulrana95** [comment]
@MattCrl We need to write a document or something?

**Comment by 94151** [comment]
@rahulrana95 i think we need to add css styles on the project by clone the repo to our local computer

**Comment by rahulrana95** [comment]
@94151 @MattCrl We can decide on css style guide so it will be better than to write css and have consistent styles

**Comment by inyutin** [comment]
Shouldn't we make all pictures added to same size?

**Comment by ardgz** [comment]
> 
> 
> Shouldn't we make all pictures added to same size?

I agree we should have a standard image size of some sort so the images look consistent. Could do with a CSS rule to set image size as well

**Comment by inyutin** [comment]
I will do standard image size now

**Comment by inyutin** [comment]
Edit here: https://github.com/MattCrl/Contributors-portraits/pull/15

**Comment by jujug12** [comment]
I would like to help on this issue. What project do you want me to add CSS style too?

**Comment by MattCrl** [comment]
I've no specific style in headfor this project. Something sober is fine ! 
I'll check PR when i have the time, i'm working alone on it so it may take some time if there's alot of PR ;)

**Comment by inyutin** [comment]
@MattCrl I'm really interested in this project, so I can help you. E.g. I can check PR with new cards and merge them :)

**Comment by yas-sisodia** [comment]
@MattCrl Please assign this issue to me, I can solve the problem which you are facing
